src/main/java/frc/robot/subsystems/intake/PassToScorer.java

```diff
@@ -58,6 +60,9 @@ public void end(boolean interrupted) {
     intakeSubsystem.setIntakeMotor(0);
     intakeSubsystem.setConveyerMotor(0);
     coralScorer.scoreCommand().end(interrupted);
+    Commands.run(() -> coralScorer.setScoreMotor(kAlignSpeed))
+        .andThen(Commands.waitSeconds(2))
+        .andThen(() -> coralScorer.stopMotor());
```

Comment:
This isn't going to run here. I would add a .andThen to the passToScorer in Robot Container